gemrb/core/DisplayMessage.h

```diff
@@ -37,7 +37,7 @@
 
 namespace GemRB {
 
-/** GUIColors options */
+/** GUIColors options. This starts at 7 because is the row in the colors.2da that corresponds to white color.*/
```

Comment:
A better comment would be that the first six colors from colors.2da are not indexed here because they are only used on the python side.